packages/dev/docs/pages/react-spectrum/styling.mdx

```diff
@@ -0,0 +1,204 @@
+```
+
+All of the available style props are listed below.
+
+<PropTable component={{
+  props: {
+    // Filter out UNSAFE properties
+    properties: Object.fromEntries(Object.entries(sharedDocs.exports.StyleProps.properties).filter(([k]) => !k.startsWith('UNSAFE_')))
+  }
+}} links={sharedDocs.links} />
+
+## Dimension values
+
+Where applicable, each style property accepts Spectrum defined variables in addition to raw CSS values.
+**Using Spectrum variables is preferred wherever possible**. These variables conform to Spectrum design
+defined sizing and spacing requirements, and also automatically adapt on different devices. For example,
+on touch screen devices, all Spectrum components expand in size to give users larger hit targets.
+
+The list of dimension values is visualized below. Use the picker to see how they change between desktop
+and mobile.
+
+```tsx import
+import {View} from '@react-spectrum/view';
+import {Flex} from '@react-spectrum/layout';
+import {Picker, Item} from '@react-spectrum/picker';
+import {Provider} from '@react-spectrum/provider';
+import {Example} from '@react-spectrum/docs/src/ThemeSwitcher';
+import {useState} from 'react';
+import dnaDocs from 'docs:@react-types/shared/src/dna.d.ts';
+
+function DimensionViewer() {
+  let [selectedKey, setSelectedKey] = useState('medium');
+
+  return (
+    <Example>
+      <Picker label="Scale" selectedKey={selectedKey} onSelectionChange={setSelectedKey}>
+        <Item uniqueKey="medium">Medium (desktop)</Item>
+        <Item uniqueKey="large">Large (mobile)</Item>
+      </Picker>
+      <Provider scale={selectedKey}>
+        <ul style={{listStyle: 'none', padding: 0, overflow: 'auto'}}>
+          {console.log(dnaDocs.links[dnaDocs.exports.DimensionValue.id])}
+          {dnaDocs.links[dnaDocs.exports.DimensionValue.id].value.elements.filter(v => v.value && !v.value.startsWith('static')).map(v =>
+            <li key={v.value}>
+              <Flex alignItems="center" marginBottom="size-25">
+                <View backgroundColor="blue-400" width={v.value} height="size-250" marginEnd="size-100" flexShrink={0} />
+                {v.value}
+              </Flex>
+            </li>
+          )}
+        </ul>
+      </Provider>
+    </Example>
+  );
+}
+
+ReactDOM.render(<DimensionViewer />, document.getElementById('dimension-viewer'));
+```
+
+<div id="dimension-viewer" />
+
+## Custom components
+
+Sometimes, you may find yourself needing to build a component that doesn't exist in Spectrum yet. In these cases,
+you can ensure consistency with other Spectrum components by utilizing existing Spectrum variables. For example,
+there are Spectrum variables for colors, border sizes, paddings, etc.
+
+These variables could be consumed in CSS directly, but if you're building something simple, you could consider
+using the `View` component from React Spectrum. `View` is like a `div` or `span`, but with additional style properties that
```

Comment:
is it like a `div` or is it like a `span`? the difference matters for display block vs display inline, we should be careful which one we say it's like for the default case

Reply:
mmm I guess I could clarify. It's both via the `elementType` prop.